tasks_management/tests/task_service_tests.py

```diff
@@ -100,3 +111,73 @@ def test_fail_task(self):
         self.assertTrue(result)
         self.assertTrue(result['success'])
         self.assertEqual(Task.objects.filter(id=obj_id).first().status, Task.Status.FAILED)
+
+    def test_resolve_task_any(self):
+        create_payload = {
+            **task_payload_resolve_any,
+            "task_group_id": self.taskgroup_any_id
+        }
+        result = self.service.create(create_payload)
+
+        self.assertTrue(result)
+        self.assertTrue(result['success'])
+        obj_id = result['data']['id']
+        self.assertTrue(Task.objects.filter(id=obj_id).exists())
+
+        resolve_payload = {"id": result["data"]["uuid"], "business_status": {"Jan Kowalski": "APPROVED"}}
```

Comment:
Why you don't use here constant value  like in Task.Status.FAILED? It should be Task.Status.APPROVED.

Reply:
It is not a task status but rather businessStatus. However I defined it in new file "utils" and use it from there.